src/core/server/base.js

```diff
@@ -125,6 +125,7 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
           includeSri: isDeployed,
           sriData,
           store,
+          trackingEnabled: config.get('trackingEnabled'),
```

Comment:
This is technically different than what the tracking library is using to enable itself (since it checks that the ID is present as well).